### PR TITLE
ADCM-3040 CEF logger for audit

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

File: python/audit/cef_logger.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    signature_id: str,
+    severity: int = 1,
+    empty_resource: bool = False,
+) -> None:
+    extension = OrderedDict.fromkeys(CEFLogConstants.extension_keys, None)
+    extension["timestamp"] = str(tz.now())
+
+    if isinstance(audit_instance, AuditSession):
+        operation_name = CEFLogConstants.operation_name_session
+        if audit_instance.user:
```
Comment (a-alferov) [inline python/audit/cef_logger.py:48]: Field `user` for `AuditSession` model could be `User` or `None`. Please add an explicit check for `None`

File: python/adcm/settings.py [hunk 2]
```diff
@@ -255,6 +256,11 @@
             'class': 'logging.FileHandler',
             'filename': os.path.join(BASE_DIR, 'data/log/adwp.log'),
         },
+        'stdout_raw': {
```
Comment (a-alferov) [inline python/adcm/settings.py:259]: I think the name `stdout` will be enough to understand

**Review by a-alferov — CHANGES_REQUESTED** [review]

**Comment by DanBalalan** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @DanBalalan! Build has been launched!

File: python/audit/cef_logger.py (hunk 1 above)
Comment (DanBalalan) [reply python/audit/cef_logger.py]: done

File: python/adcm/settings.py (hunk 2 above)
Comment (DanBalalan) [reply python/adcm/settings.py]: done

File: python/audit/cef_logger.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        if audit_instance.user is not None:
+            extension["actor"] = audit_instance.user.username
+        else:
+            extension["actor"] = audit_instance.login_details.get("username", "<undefined>")
+        extension["operation"] = operation_name
+        extension["result"] = audit_instance.login_result
+
+    elif isinstance(audit_instance, AuditLog):
+        operation_name = audit_instance.operation_name
+        extension["actor"] = audit_instance.user.username
```
Comment (a-alferov) [inline python/audit/cef_logger.py:57]: User can be `None`

**Review by a-alferov — CHANGES_REQUESTED** [review]

**Comment by a-alferov** [comment]
Remove pls `stdout` from `django` handler https://github.com/arenadata/adcm/pull/1966/files#diff-bf96bca58a79732d762817b4394ce90028168055de949401fb90260a5d58fb03R267

File: python/audit/cef_logger.py (hunk 3 above)
Comment (DanBalalan) [reply python/audit/cef_logger.py]: done

**Review by a-alferov — APPROVED** [review]

**Pull request merged**